plugins/broker-redis/.amplicationrc.json

```diff
@@ -1,6 +1,7 @@
 {
     "settings": {
-        "url": "redis://localhost:6379",
+        "host": "localhost",
+        "port": 6379,
```

Comment:
Are you sure that it works with our NestJS version (and not the latest)

Reply:
@d-sonuga

Reply:
@abrl91, yes I'm sure it works with Nestjs v8. I changed `url` to `host` and `port` because the docker-compose .yml files need the host and port to be properly configured. The URL to be used in the Nestjs Redis microservice setting is built from the user-specified host and port.